app/(info-page)/_components/PersonCard.tsx

```diff
@@ -0,0 +1,39 @@
+import Image from 'next/image'
+
+type StaticImageData = {
+  src: string
+  height: number
+  width: number
+  blurDataURL?: string
+  priority?: string
+}
+
+type PersonType = {
+  title: string
+  desc: string
+  image: string | StaticImageData
+}
+
+const PersonCard = ({ title, desc, image }: PersonType) => {
+  return (
+    <div className='flex flex-col items-center gap-8 pt-12 md:flex-row md:pt-16'>
```

Comment:
you're using both padding and gap here! i would suggest keeping the gap and moving the padding to the title since now every PersonCard gets the padding and it should only be on the first one / the title

Reply:
then you have to change it in the wrapper component! right now there's too much spacing. remove it here and add it to the title in the wrapper component